Updates/Instances/540_shattered_halls.sql

```diff
@@ -597,9 +607,12 @@ INSERT INTO `creature_spawn_entry` (`guid`, `entry`) VALUES
 (@CGUID+136, 16523), (@CGUID+136, 17420), -- Shattered Hand Savage, Shattered Hand Heathen
 (@CGUID+137, 16523), (@CGUID+137, 17420), -- Shattered Hand Savage, Shattered Hand Heathen
 (@CGUID+138, 16523), (@CGUID+138, 17420), -- Shattered Hand Savage, Shattered Hand Heathen
--- (@CGUID+140, 16594), (@CGUID+140, 17420), -- Shadowmoon Acolyte, Shattered Hand Heathen
--- (@CGUID+141, 16594), (@CGUID+141, 17420), -- Shadowmoon Acolyte, Shattered Hand Heathen
```

Comment:
When i added this commented like that it's confirmed there is also chance for 16594 Shadowmoon Acolyte to be at these locations, i just didnt want to add them as it would be fully randomzied what spawns would be up due to how creature_spawn_entry currently works

Reply:
Yeah its realy wierd how they get spawned. Only thing missing here is switching the position of heaven/savage and one of the 3. 
Dont know how to do this. Maybe with pools?

Reply:
Idk i'd like to keep all potential spawns until we figure out how they are varied

Reply:
Yeah im gonna change it. I figuered out how to do it correctly.

Reply:
Should also be fixed now with pools.